src/commands/Tools/Dictionary/define.ts

```diff
@@ -0,0 +1,82 @@
+import { KlasaMessage } from 'klasa';
+import { TOKENS } from '@root/config';
+import { DbSet } from '@lib/structures/DbSet';
+import { RichDisplayCommand, RichDisplayCommandOptions } from '@lib/structures/RichDisplayCommand';
+import { UserRichDisplay } from '@lib/structures/UserRichDisplay';
+import { cutText, fetch, FetchResultTypes } from '@utils/util';
+import { toTitleCase } from '@klasa/utils';
+import { MessageEmbed } from 'discord.js';
+import { ApplyOptions } from '@skyra/decorators';
+import { BrandingColors } from '@utils/constants';
+
+const options = { headers: { Accept: 'application/json', Authorization: `Token ${TOKENS.OWLBOT}` } };
+
+@ApplyOptions<RichDisplayCommandOptions>({
+	aliases: ['definition', 'defination', 'dictionary'],
+	bucket: 2,
+	cooldown: 20,
+	description: language => language.get('COMMAND_DEFINE_DESCRIPTION'),
+	extendedHelp: language => language.get('COMMAND_DEFINE_EXTENDED'),
+	usage: '<input:string>'
+})
+export default class extends RichDisplayCommand {
+
+	public async run(message: KlasaMessage, [input]: [string]) {
+		const response = await message.sendEmbed(new MessageEmbed()
+			.setDescription(message.language.tget('SYSTEM_LOADING'))
+			.setColor(BrandingColors.Secondary));
+
+		try {
+			const result = await fetch<OwlbotResultOk>(
+				`https://owlbot.info/api/v4/dictionary/${encodeURIComponent(input.toLowerCase())}`,
+				options,
+				FetchResultTypes.JSON
+			);
+
+			const display = await this.buildDisplay(result, message);
+			await display.start(response, message.author.id);
+			return response;
+		} catch {
+			throw message.language.get('COMMAND_DEFINE_NOTFOUND');
+		}
```

Comment:
Move the fetch to a fetchApi function because rn api errors affect message sending. 

```ts
const result = await this.fetchApi(...);
```